js/search/utils/search-launchd.js

```diff
@@ -0,0 +1,105 @@
+const { exec } = require('child_process');
+const os = require('os');
+const { randomString } = require('./randomString.js');
+const log = require('../../log.js');
+const logLevels = require('../../enums/logLevels.js');
+const Winreg = require('winreg');
+
+/**
+ * Register for creating launch agent
+ * @type {Registry}
+ */
+const regKey = new Winreg({
+    hive: Winreg.HKCU,
+    key: '\\Software\\Microsoft\\Windows\\CurrentVersion\\Run'
```

Comment:
Is this the right location in the registry?

Reply:
Also, is there no way to avoid using the registry?

Reply:
Yes, the location is correct.
No, this is the only option.